src/add-hooks.coffee

```diff
@@ -95,23 +94,43 @@ addHooks = (runner, transactions, callback) ->
         callback(new Error(msg))
     else
 
-      # no data found, doing nothing
+      # No data found, doing nothing
       return callback()
 
   # Loading hookfiles from fs
   else
+    # Expand hookfiles - sort files alphabetically and resolve their paths
+    hookfiles = [].concat runner.configuration?.options?.hookfiles
+    files = hookfiles.reduce((result, item) ->
```

Comment:
Addressed in https://github.com/apiaryio/dredd/pull/803/commits/60d2a4fdce00d10dccc66cfbca5d7c0bbe11be8e.